src/screens/doc/DocIconsScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../../components';
+import { colors, Icon, normalize } from '../../config';
+
+type TIconItem = {
+  iconName: keyof typeof Icon;
+};
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+
+  return (
+    <View style={styles.iconContainer}>
+      <SelectedIcon size={normalize(35)} color={colors.secondary} />
+      <RegularText smallest>{iconName}</RegularText>
+    </View>
+  );
+};
+
+export const DocIconsScreen = () => {
+  return (
+    <FlatList
+      data={Object.keys(Icon).filter((name) => name !== "NamedIcon")}
```

Comment:
![image](https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/acafc737-f81b-43ac-b131-6f7f7dddfd48)

Reply:
Adjusted it!

Reply:
Where? I cannot see it.

Reply:
ah i see, please do not comment on local development, as we cannot follow it.